lib/IRGen/GenDecl.cpp

```diff
@@ -3152,7 +3151,7 @@ IRGenModule::getAddrOfGlobalConstantString(StringRef utf8) {
 
   // FIXME: Big endian-ness.
```

Comment:
You can't break all those 32-bit big endian platforms... we still care about m68k right?

Reply:
If it's not in the tests it doesn't exist.

Having said that the comment no longer applies; any endianness problems would now be inside InlineRefCountBits. I'll remove it.

Reply:
Removed.